src/styles/navlist.scss

```diff
@@ -0,0 +1,45 @@
+.nav-list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+
+  &__item {
+    margin-right: 20px;
+    list-style-type: none;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    line-height: 60px;
+    font-weight: 500;
+    display: block;
+    color: $base-color;
+    font-size: 12px;
+    text-transform: uppercase;
+    text-decoration: none;
+    position: relative;
+
+    &:hover {
+      color: $blue-color;
+    }
+  }
+}
+
+.is-active {
+  color: $blue-color;
```

Comment:
The variable `$blue-color` is used here but is not defined in this file. Ensure that it is defined in a global SCSS file or imported correctly to avoid compilation errors.